### PR TITLE
bumped up 7 series to 7.74.0.Final

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: data/pom.yml [hunk 1]
```diff
@@ -27,59 +27,59 @@ latestFinal:
...
-    documentationHtmlSingle: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.drools.org/7.73.0.Final/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.drools.org/7.74.0.Final/kie-api-javadoc/index.html
 
-    droolsWhatsNew: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
-    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391094
+    droolsWhatsNew: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
+    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=new_releasenotes_number
```
Comment (tarilabs) [inline data/pom.yml:82]:
```suggestion
    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391774
```

File: data/pom.yml [hunk 2]
```diff
@@ -27,59 +27,59 @@ latestFinal:
...
```
Comment (tarilabs) [inline data/pom.yml:51]:
```suggestion
    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391774
```

**Review by tarilabs — CHANGES_REQUESTED** [review]
I believe the `version=` number shall be reflected to show 12391774

**Comment by mbiarnes** [comment]
@tarilabs good spot! There was missing a `$` in the update-version script

**Review by tarilabs — APPROVED** [review]

File: data/pom.yml [hunk 3]
```diff
@@ -27,59 +27,59 @@ latestFinal:
...
-    documentationHtmlSingle: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.drools.org/7.73.0.Final/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.drools.org/7.74.0.Final/kie-api-javadoc/index.html
 
-    droolsWhatsNew: https://docs.drools.org/7.73.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
-    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391094
+    droolsWhatsNew: https://docs.drools.org/7.74.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
+    droolsReleaseNotesV7: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12391774
```
Comment (danielezonca) [inline data/pom.yml:51]: What about adding an alert-link in the home page too?
https://github.com/kiegroup/drools-website/blob/main/templates/index.ftl#L17

Something like
```html
            <a class="alert-link"
            href="${pom.latestFinal7.droolsReleaseNotesV7}">Drools ${pom.latestFinal7.version} with bugfixes has been released.</a>
```

Comment (mbiarnes) [reply data/pom.yml:51]: @danielezonca maybe on the download page? If we put it on main page we will have always two alert-links

Comment (danielezonca) [reply data/pom.yml:51]: I have no strong opinion, we can remove it when we want of course.
The download page has already the information as far as I know

Comment (tarilabs) [reply data/pom.yml:51]: done with c6c3d90

Comment (tarilabs) [reply data/pom.yml:51]: Screenshot 2023-06-30 at 09 06 20

**Pull request merged**